Tribler/community/tunnel/crypto/tunnelcrypto.py

```diff
@@ -99,9 +104,11 @@ def is_key_compatible(self, key):
     def generate_diffie_secret(self):
         return '', ''
 
+    # TODO delete? returns tuple of 3 empty things, doesn't even call generate_diffie_shared_secret(x,y,z)
     def generate_diffie_shared_secret(self, dh_received):
```

Comment:
This is the NoCrypto implementation, it allows you to turn off crypto :smile: Hence the empty stuffs